### PR TITLE
Fixed #1191 All info logs changed to debug except for initial startup…

### LINKED ISSUE #1191: Change configstore reporting to be DEBUG other than first setup

### PR DESCRIPTION
**Pull request by craig8**
… message.

Changed Unreachable to be warning rather than debug.
Also modified to be closer to pep-8 standards.

---
This change is [Reviewable](https://reviewable.io/reviews/volttron/volttron/1199)

### CODE REVIEW

File: volttron/platform/store.py [hunk 1]
```diff
@@ -197,18 +197,19 @@ def manage_delete_store(self, identity):
...
         with agent_store_lock:
             try:
-                self.vip.rpc.call(identity, "config.update", "DELETE_ALL", None, trigger_callback=True).get(
-                    timeout=10.0)
+                self.vip.rpc.call(identity, "config.update",
+                                  "DELETE_ALL", None,
+                                  trigger_callback=True).get(timeout=10.0)
             except errors.Unreachable:
-                _log.debug("Agent {} not currently running. Configuration update not sent.".format(identity))
+                _log.warn("Agent {} not currently running. Configuration update not sent.".format(identity))
```
Comment (kmonson) [inline volttron/platform/store.py]: This should stay as a debug statement or maybe info as it's common and normal.

File: volttron/platform/store.py [hunk 2]
```diff
@@ -221,7 +222,7 @@ def manage_list_configs(self, identity):
     @RPC.export
     def manage_list_stores(self):
         identity = bytes(self.vip.rpc.context.vip_message.peer)
-        _log.info("manage_list_stores called by {}".format(identity))
+        _log.debug("manage_list_stores called by {}".format(identity))
```
Comment (kmonson) [inline volttron/platform/store.py]: This can go away.

File: volttron/platform/store.py [hunk 3]
```diff
@@ -278,27 +284,30 @@ def get_configs(self):
 
         with agent_store_lock:
             try:
-                self.vip.rpc.call(identity, "config.initial_update", agent_configs).get(timeout=10.0)
+                self.vip.rpc.call(identity, "config.initial_update",
+                                  agent_configs).get(timeout=10.0)
             except errors.Unreachable:
-                _log.debug("Agent {} not currently running. Configuration update not sent.".format(identity))
+                _log.warn("Agent {} not currently running. Configuration update not sent.".format(identity))
```
Comment (kmonson) [inline volttron/platform/store.py]: This should stay as a debug statement or maybe info as it's common and normal.

File: volttron/platform/store.py [hunk 4]
```diff
@@ -321,28 +330,30 @@ def delete(self, identity, config_name, trigger_callback=False, send_update=True
...
+        # Sync will delete the file if the store is empty.
         agent_disk_store.async_sync()
 
         if send_update:
             with agent_store_lock:
                 try:
                     self.vip.rpc.call(identity, "config.update", "DELETE", config_name, trigger_callback=trigger_callback).get(timeout=10.0)
                 except errors.Unreachable:
-                    _log.debug("Agent {} not currently running. Configuration update not sent.".format(identity))
+                    _log.warn("Agent {} not currently running. Configuration update not sent.".format(identity))
```
Comment (kmonson) [inline volttron/platform/store.py]: This should stay as a debug statement or maybe info as it's common and normal.

File: volttron/platform/store.py [hunk 5]
```diff
@@ -394,14 +409,14 @@ def _add_config_to_store(self, identity, config_name, raw, parsed, config_type,
...
-        _log.info("Agent {} config {} stored.".format(identity, config_name))
+        _log.debug("Agent {} config {} stored.".format(identity, config_name))
 
         if send_update:
             with agent_store_lock:
                 try:
                     self.vip.rpc.call(identity, "config.update", action, config_name, contents=parsed, trigger_callback=trigger_callback).get(timeout=10.0)
                 except errors.Unreachable:
-                    _log.debug("Agent {} not currently running. Configuration update not sent.".format(identity))
+                    _log.warn("Agent {} not currently running. Configuration update not sent.".format(identity))
```
Comment (kmonson) [inline volttron/platform/store.py]: This should stay as a debug statement or maybe info as it's common and normal.

File: volttron/platform/store.py (hunk 1 above)
Comment (craig8) [inline volttron/platform/store.py]: Addressed

File: volttron/platform/store.py (hunk 2 above)
Comment (craig8) [inline volttron/platform/store.py]: Addressed

File: volttron/platform/store.py (hunk 4 above)
Comment (craig8) [inline volttron/platform/store.py]: Addressed

File: volttron/platform/store.py (hunk 5 above)
Comment (craig8) [inline volttron/platform/store.py]: Addressed

**Pull request merged**